src/d3a/d3a_core/sim_results/area_statistics.py

```diff
@@ -122,7 +127,18 @@ def _is_cell_tower_node(area):
 
 
 def _is_load_node(area):
-    return isinstance(area.strategy, LoadHoursStrategy)
+    return isinstance(area.strategy, (LoadHoursStrategy, CellTowerLoadHoursStrategy,
```

Comment:
No need fo the CellTowerLoadHoursStrategy, isinstance is capable of detecting whether subclasses originate from a base class. Since CellTower inherits from LoadHours, it should detect it.